Source/JavaScriptCore/yarr/RegularExpression.cpp

```diff
@@ -38,29 +37,25 @@ namespace JSC { namespace Yarr {
 
 class RegularExpression::Private : public RefCounted<RegularExpression::Private> {
 public:
-    static Ref<Private> create(StringView pattern, TextCaseSensitivity caseSensitivity, MultilineMode multilineMode, UnicodeMode unicodeMode)
+    static Ref<Private> create(StringView pattern, OptionSet<JSC::Yarr::Flags> flags)
```

Comment:
You shouldn't need to use the Yarr prefix anywhere here since you're already in that namespace -- I'm surprised there's so much existing code doing so.